src/lib/spinel/openthread-spinel-config.h

```diff
@@ -55,4 +55,14 @@
 #define OPENTHREAD_SPINEL_CONFIG_RCP_RESTORATION_MAX_COUNT 0
 #endif
 
+/**
+ * @def OPENTHREAD_SPINEL_CONFIG_RESET_CONNECTION
+ *
+ * Define 1 to reset connection after hard resetting RCP(USB CDC ACM) device.
+ *
+ */
+#ifndef OPENTHREAD_SPINEL_CONFIG_RESET_CONNECTION
+#define OPENTHREAD_SPINEL_CONFIG_RESET_CONNECTION 0
```

Comment:
Can we remove this configuration and just use `OPENTHREAD_PLATFORM_USE_PSEUDO_RESET` to determine if we need to reset connection?

Reply:
> Can we remove this configuration and just use `OPENTHREAD_PLATFORM_USE_PSEUDO_RESET` to determine if we need to reset connection?


We cannot do that. We don't want to change behaviour for "true" UART communication for which `OPENTHREAD_PLATFORM_USE_PSEUDO_RESET` is disabled as well.

Reply:
Sgtm